curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-assert.isok-and-assert.isnotok.english.md

```diff
@@ -8,12 +8,12 @@ challengeType: 2
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
 
+<code>isOk(truthy)</code> and <code>isNotOk(falsey)</code> will pass.
```

Comment:
This sentence does not make sense on its own.

Reply:
I've changed these lines to provide a little more context.